### PR TITLE
fix #72828 and #73018

### LINKED ISSUE #72828: kisahdunia.com
### Issue URL (Anti Adblock Script)
 [https://kisahdunia.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fkisahdunia.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (60), <br/>Block Push API, <br/>Block Location API
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS<br/><br/><b>Language-specific:</b><br/> ABPindo, <br/>AdBlockID
Custom filter subscriptions: | https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances.txt,<br>https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/privacy.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#72828 and #73018

### CODE REVIEW

**Comment by wout-codes** [comment]
@zloyden Please take a look.

**Comment by wout-codes** [comment]
@zloyden Please take a look.

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -510,11 +510,11 @@ freegogpcgames.com,omgpu.com,arcadepunks.com#%#//scriptlet("set-constant", "DHAn
...
-graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet("set-constant", "adsbygoogle.length", "undefined")
-||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$script,xmlhttprequest,redirect=googlesyndication-adsbygoogle,domain=graphicux.com|w3layouts.com|musike.net|windowsmatters.com|sonixgvn.net|freeiptvlists.com|avoiderrors.com|gatcha.org|sayrodigital.com|justandroid.net|4horlover.com|topnewsshow.com|3dzip.org|ovagames.com
+kisahdunia.com,graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet('prevent-addEventListener', 'load', 'daadb_get_data')
+kisahdunia.com,graphicux.com,w3layouts.com,musike.net,windowsmatters.com,sonixgvn.net,freeiptvlists.com,avoiderrors.com,gatcha.org,sayrodigital.com,justandroid.net,4horlover.com,topnewsshow.com,3dzip.org,ovagames.com#%#//scriptlet("set-constant", "adsbygoogle.length", "undefined")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:514]: I don't find those scriptlets useful. 
1. Remove `kisahdunia.com` from common rules.
2. Add rules below with task link in this section.
```
! https://github.com/AdguardTeam/AdguardFilters/issues/72828
||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$important,script,xmlhttprequest,redirect=googlesyndication-adsbygoogle,domain=kisahdunia.com
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=kisahdunia.com
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt:514]: Where should I exactly put the rules? There is a common section for the rules you suggested.

Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt:514]: It's a little lower after `! END: Ads replacement`.

Comment (wout-codes) [reply EnglishFilter/sections/antiadblock.txt]: Now?

**Review by zloyden — APPROVED** [review]

File: EnglishFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply EnglishFilter/sections/antiadblock.txt]: I changed it a bit, it's ok now.

**Pull request merged**